ignite/cli
# feat: merge the swagger files manually instead use nodetime `swagger-combine`

## Description

This PR changes the package swagger-combine to merge all swagger files into the code instead of using the NodeJs pkg `swagger-combine` inside the nodetime binary.

### How to test

- Scaffold and run a chain:
```shell
ignite s chain venus && cd venus && \
ignite s list --yes user email && \
ignite c serve
```

- Check the swagger files at http://0.0.0.0:1317
